### DISCUSSION TITLE
version 2.4.6 under Linux Mint 18

### DISCUSSION THREAD

**Discussion by RickGroszkiewicz** [artisan-roaster-scope/artisan#719] [Q&A] [answered]
I recently tried to install artisan 2.4.6. It does not appear to start properly. I am using Linux Mint 18.

**Comment by kspaans** [comment, marked as answer]
I had the same problem in Ubuntu 16.04, which had a version of `glibc` that was too old. You will likely need to update your Linux Mint.

**Reply by RickGroszkiewicz** [reply]
Hmmmmm, I did get the "End of Life" notice about version 18 of Linux Mint. I'll go back to roasting with version 2.4.4 for now.

I will try artisan 2.4.6 with a newer version of Linux on another laptop.

**Reply by RickGroszkiewicz** [reply]
Turns out artisan 2.4.4 does not quite work properly in version 18 of Linux Mint. After looking carefully at my settings files, I was using artisan 2.4.2 instead.

**Comment by RickGroszkiewicz** [comment]
I finally documented ALL the changes to settings I made for artisan 2.4.6 under Linux Mint 20. This photo gallery shows the changes for my old TC4 Hottop first, then the changes for the Hottop B-2K+
"http://www.softwarepolish.com/personal/Artisan-Reinstalled-Linux-Mint-20/album/index.html"
